app/models/manageiq/providers/embedded_automation_manager.rb

```diff
@@ -1,10 +1,5 @@
 class ManageIQ::Providers::EmbeddedAutomationManager < ManageIQ::Providers::AutomationManager
-  require_nested :Authentication
-  require_nested :ConfigurationScript
-  require_nested :ConfigurationScriptPayload
-  require_nested :ConfigurationScriptSource
-  require_nested :ConfiguredSystem
-  require_nested :OrchestrationStack
+  require_nested_all __FILE__
```

Comment:
I think this should take __dir__ as a parameter instead, or even better no parameter at all, and just determine it from the caller.

Reply:
haha, ok, now pasted and see the `byebug-10...`...
In byebug session, the caller is different from actual runtime. so the `caller(1, 1)` will work.
thanks @Fryguy